cmd/arc/main.go

```diff
@@ -219,7 +219,7 @@ func getFormat(subcommand string) (interface{}, error) {
 		v.ContinueOnError = continueOnError
 		v.Password = os.Getenv("ARCHIVE_PASSWORD")
 	case *archiver.Tar:
-		v = mytar
+		f = mytar
```

Comment:
@mholt I'd like your comment on this, please.

Before it was a no-op. Now it's acting as intended, and the tests are passing, but I'm not sure of any potential consequences. I'd be happy to revert this if there's any concern.

Reply:
This is fine, thanks! I think it's effectively equivalent since the value is a pointer.